### DISCUSSION TITLE
unable to install bluedot on raspberry pi 5

### DISCUSSION THREAD

**Discussion by Laxin1234** [community/community#122306] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

unable to install bluedot on raspberry pi 5 by using sudo pip3 install bluedot
it shows:error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    For more information visit http://rptl.io/venv

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

even if I us sudo apt install or sudo apt get it shows Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package bluedot
what should I do?

**Comment by Garden999** [comment]
Hi, has your issue got solved? I met the same one and am not sure if the Bluebot package is still avaible in Python3.
